### ISSUE TITLE
Training on VOC2011

### ISSUE THREAD

**Issue by mat112** [JihongJu/keras-fcn#25]
The README suggests that we should be able to train on VOC2011 just by downloading the dataset and running train.py. Is it really the case? My training seems to converge for the first few epochs, but  then the val_loss stops to improve early, and diverge from training loss. In fact the best val_loss I can have is around 1.06. Do you have an idea why?

Thank you!

Epoch 14/100
1112/1112 [==============================] - 878s - loss: 1.0603 - acc: 0.7734 - val_loss: 1.1059 - val_acc: 0.7623
Epoch 15/100
1112/1112 [==============================] - 880s - loss: 1.0474 - acc: 0.7751 - val_loss: 1.0961 - val_acc: 0.7652
Epoch 16/100
1112/1112 [==============================] - 869s - loss: 1.0273 - acc: 0.7784 - val_loss: 1.1116 - val_acc: 0.7609
Epoch 17/100
1112/1112 [==============================] - 869s - loss: 1.0228 - acc: 0.7781 - val_loss: 1.1651 - val_acc: 0.7596
Epoch 18/100
1112/1112 [==============================] - 869s - loss: 1.0054 - acc: 0.7812 - val_loss: 1.1100 - val_acc: 0.7643
Epoch 19/100
1112/1112 [==============================] - 869s - loss: 0.9971 - acc: 0.7834 - val_loss: 1.1266 - val_acc: 0.7609
Epoch 20/100
1112/1112 [==============================] - 869s - loss: 0.9881 - acc: 0.7833 - val_loss: 1.1472 - val_acc: 0.7581
[...]
Epoch 44/100
1112/1112 [==============================] - 869s - loss: 0.6450 - acc: 0.8553 - val_loss: 1.2859 - val_acc: 0.7561
Epoch 45/100
1112/1112 [==============================] - 868s - loss: 0.6358 - acc: 0.8582 - val_loss: 1.2139 - val_acc: 0.7645
Epoch 46/100
1112/1112 [==============================] - 869s - loss: 0.6012 - acc: 0.8688 - val_loss: 1.3206 - val_acc: 0.7573
Epoch 47/100
1112/1112 [==============================] - 868s - loss: 0.5956 - acc: 0.8704 - val_loss: 1.2663 - val_acc: 0.7626

**Comment by JihongJu** [comment]
@mat112 Thank you. I had the same problem when I trained and that is why I don't provide a trained model. 

To me, this is a typical overfitting issue and it should be solvable by tuning the regularization term. I am currently busy with other things and don't have the time to tune it. Let me know if you cannot find a proper beta for regularization.

**Comment by Pan-zhaoyu** [comment]
Ｔhank you for your reply. I have solved the above issue. Since I am a beginner. Now I wonder to know how can I test my own picture using the model and the existing weights in this program? I will appreciate it if you could give me some advice or idea. THx!

**Comment by hwei-hw** [comment]
@Pan-zhaoyu Thank you. I have met the overfitting issue in train the network. And could you tell me how to fix the issue? I will very appreciate it if you could give me some advice. Thanks very much!

**Comment by kangmengmeng** [comment]
@Pan-zhaoyu @Atomwh @JihongJu   How do you slove the overfitting issue?    O_O   Thanks！
